controllers/updateservice_controller.go

```diff
@@ -350,8 +362,74 @@ func (r *UpdateServiceReconciler) ensureTrustedClusterCA(ctx context.Context, re
 	return err
 }
 
+
+func (r *UpdateServiceReconciler) ensureGraphDataSHA(ctx context.Context, reqLogger logr.Logger, instance *cv1.UpdateService) (string, error){
+
+	if strings.Contains(instance.Spec.GraphDataImage, "@sha256"){
+		return "", nil
+	}
+
+	pod := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "graph-data-tag-digest",
+			Namespace: instance.Namespace,
+			Annotations: map[string]string{
+				"updateservice.operator.openshift.io/last-refresh":time.Now().UTC().Format(time.RFC822),
+			},
+		},
+		Spec: corev1.PodSpec{
+			RestartPolicy: corev1.RestartPolicyNever,
+			Containers: []corev1.Container{
+				{
+					Name:            NameInitContainerGraphData,
+					Image:           instance.Spec.GraphDataImage,
+					ImagePullPolicy: corev1.PullAlways,
+					Command: 		 []string{"/bin/sh", "-c", "--"},
+					Args: 			 []string{"sleep 300;"},
+				},
+
+			},
+		},
+	}
+
+	if err := controllerutil.SetControllerReference(instance, pod, r.Scheme); err != nil {
+		return "", err
+	}
+
+	// Check if this pod already exists
+	found := &corev1.Pod{}
+	err := r.Client.Get(ctx, types.NamespacedName{Name: pod.Name, Namespace: pod.Namespace}, found)
+
+	if err != nil && apiErrors.IsNotFound(err) {
+		reqLogger.Info("Creating Pod", "Namespace", pod.Namespace, "Name", pod.Name)
+		err := r.Client.Create(ctx, pod)
+		if err != nil {
+			handleErr(reqLogger, &instance.Status, "CreateGraphDataPodFailed", err)
+		}
+		return "", err
+	} else if err != nil {
+		handleErr(reqLogger, &instance.Status, "GetGraphDataPodFailed", err)
+		return "", err
+	} else if found.Status.Phase == "Succeeded" {
+		err := r.Client.Delete(ctx, pod)
+		if err != nil {
+			handleErr(reqLogger, &instance.Status, "CreateGraphDataPodFailed", err)
+		}
+		return "", err
+	}
+
+	if len(found.Status.ContainerStatuses) > 0 {
+		return found.Status.ContainerStatuses[0].ImageID, nil
+	}else{
```

Comment:
Huh does this pass `go fmt`? I think we may be missing a `go fmt`-enforcing CI job

Reply:
https://github.com/openshift/release/pull/36878